automatic_generation.py

```diff
@@ -0,0 +1,105 @@
+import subprocess
+import cv2
+from simple_term_menu import TerminalMenu
+# pylint: disable-next=import-error
+from TTS.api import TTS
+import reddit_integration as ri
+import utils
+import tts
+
+BACKGROUND_VIDEO_PATH = './video-generation/public/video/backgroundVideo.mp4'
+OUTPUT_FILE_BUZZ_WORDS = 'reddit_story_crazy_reading_reddit_short_funny_sad_insane'
+RUN_OPTIONS = ['One', 'Specify number']
+
+def preview():
+    generate_videos(number_of_videos=1, run_in_preview_mode=True)
+
+def default():
```

Comment:
I assume default only runs after you have selected 1 or 0, and you can't select anything else?